gen/generator.jl

```diff
@@ -19,43 +19,49 @@ options = load_options(joinpath(@__DIR__, "generator.toml"))
 # append!(options["general"]["output_ignorelist"], extract_inlined_funcs(headers))
 
 # Add compiler flags
-args = get_default_args()
+args = get_default_args("x86_64-w64-mingw32")
 push!(args, "-Iheaders")
 push!(args, "-std=c99")
 
-include_dir = joinpath(open62541_jll.artifact_dir, "include") |> normpath
+# fetch Windows JLL and config macro defines
+push!(args, "-D_WIN32")
+include_dir = Generators.JLLEnvs.get_pkg_include_dir(open62541_jll, "x86_64-w64-mingw32") |> normpath
+@assert !isempty(include_dir)
+push!(args, "-I$include_dir")
 
 #copy header files to new directory
-Base.Filesystem.cptree(include_dir, "headers", force = true)
-chmod("headers", 0o777; recursive = true)
-headers = String[]
-for (root, dirs, files) in walkdir("headers")
-    for file in files
-        push!(headers, joinpath(root, file)) # path to files
-    end
-end
-headers = filter(x -> endswith(x, ".h"), headers) #just in case there are non .h files around...
-
-#comment out two lines in util.h; 
+# Base.Filesystem.cptree(include_dir, "headers", force = true)
+# chmod("headers", 0o777; recursive = true)
+# headers = String[]
+# for (root, dirs, files) in walkdir("headers")
+#     for file in files
+#         push!(headers, joinpath(root, file)) # path to files
+#     end
+# end
+# headers = filter(x -> endswith(x, ".h"), headers) #just in case there are non .h files around...
+
+#comment out two lines in util.h;
 #these caused errors since open62541_jll is compiled without amalgamation (reason not clear to me)
-fn = joinpath(@__DIR__, "./headers/open62541/common.h")
-f = open(fn, "r")
-util_content = read(f, String)
-close(f)
-orig = "struct UA_ServerConfig;
-typedef struct UA_ServerConfig UA_ServerConfig;"
-new = "//struct UA_ServerConfig;
-    //typedef struct UA_ServerConfig UA_ServerConfig;"
-util_content = replace(util_content, orig => new)
-f = open(fn, "w")
-write(f, util_content)
-close(f)
+# fn = joinpath(@__DIR__, "./headers/open62541/common.h")
+# f = open(fn, "r")
+# util_content = read(f, String)
+# close(f)
+# orig = "struct UA_ServerConfig;
+# typedef struct UA_ServerConfig UA_ServerConfig;"
+# new = "//struct UA_ServerConfig;
+#     //typedef struct UA_ServerConfig UA_ServerConfig;"
+# util_content = replace(util_content, orig => new)
+# f = open(fn, "w")
+# write(f, util_content)
+# close(f)
```

Comment:
if there is an audit error, we can use `no_audit` flag to skip it. (but it may generate invalid code)